lib/pages/new_group/new_group_chat_info.dart

```diff
@@ -95,26 +92,16 @@ class NewGroupChatInfo extends StatelessWidget {
         child: ValueListenableBuilder<Either<Failure, Success>?>(
           valueListenable: newGroupController.createRoomStateNotifier,
           builder: (context, value, child) {
-            if (value == null) {
-              return child!;
-            } else {
-              return value.fold(
-                (failure) => child!,
-                (success) {
-                  if (success is CreateNewGroupChatLoading) {
-                    return const TwakeFloatingActionButton(
-                      customIcon: SizedBox(child: CircularProgressIndicator())
-                    );
-                  } else {
-                    return const SizedBox();
-                  }
-                });
+            if (value?.isRight() == true) {
```

Comment:
What happens if `uploadAvatar` failed? Stop the process of creating room?

Reply:
it will show create button in normal state, then user can press on it to create room again